core/domain/classifier_services.py

```diff
@@ -359,9 +458,8 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
-                                               state_name):
-    """Gets the classifier object from the exploration attributes.
+def get_classifier_training_job(exp_id, exp_version, state_name):
```

Comment:
@seanlip We can't use the get-multi version for now. We will require it only when we are retrieving all the classifiers to pass to the front end. So, don't you think its better to implement it when we actually use it?